docs/content/manual/rancher-integration/drain-using-rancher-ui.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Drain using Rancher UI
+---
+
+Test with Longhorn default setting of 'Node Drain Policy': `block-if-contains-last-replica`
+----
+
+### 1. Drain operation on single node using Rancher UI
+
+**Given**  Single node cluster with Longhorn installed
+
+AND few RWO and RWX volumes attached with node/pod exists
+
+AND 1 RWO and 1 RWX volumes unattached
+
+**When** Drain the node with default values of Rancher UI
+
+**Then** Drain should be blocked
+
+AND Volumes should be healthy and data should be intact after drain completes
+
+AND The volumes which were attached initially should be attached and the ones which is unattached should be unattached
+
+AND Logs should not have any unexpected errors
+
+
+### 2. Drain operation on a multiple node using Rancher UI - default value
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND few RWO and RWX volumes attached with node/pod exists
+
+AND few RWO and RWX volumes unattached
+
+**When** Drain the worker node one by one with default values of Rancher UI
+
+**Then** Drain should be successful
+ 
+AND Volumes should be healthy and data should be intact after drain completes
+
+AND The volumes which were attached initially should be attached and the ones which is unattached should be unattached
+
+AND Logs should not have any unexpected errors
+
+
+### 3. Drain operation on a multiple node using Rancher UI - Force
+
+**Given**  Cluster with 1 etcd/control plane and 3 worker nodes with Longhorn installed
+
+AND few RWO and RWX volumes attached with node/pod exists
+
+AND few RWO and RWX volumes unattached
+
+**When** Drain the worker node one by one with Force option enabled using Rancher UI.
+
+**Then** Drain should be successful
+
+AND Volumes should be healthy and data should be intact after drain completes
+
+AND The volumes which were attached initially should be attached and the ones which is unattached should be unattached
+
+AND Logs should not have any unexpected errors.
+
```

Comment:
In Rancher v2.7.1 + Longhorn v1.3.3-rc2, I need enable both `Delete Empty Dir Data` and `Force` so the drain node will complete, if only enable `Force` checkbox, the drain will never complete, should we update steps?